src/site/content/en/blog/storage-foundation/index.md

```diff
@@ -0,0 +1,374 @@
+  what IDBFS can guarantee.
+
+## What is the Storage Foundation API? {: #what }
+
+There are two main parts to the API:
+
+- **File system calls**, which provide basic functionality to interact with files and file paths.
+- **File handles**, which provide read and write access to an existing file.
+
+### File system calls
+
+The Storage Foundation API introduces a new interface `storageFoundation` that lives on the `window`
+object and that includes a number of functions:
+
+{% Aside %} We are currently exploring the tradeoffs between providing a synchronous vs. an
+asynchronous API. The interfaces are designed to be asynchronous as a temporary measure and will be
+updated once a decision has been reached. For more background on the tradeoffs, see the
+[Explainer](https://github.com/WICG/storage-foundation-api-explainer#sync-vs-async). {% endAside %}
+
+- `storageFoundation.open(name)`: Opens the file with the given name if it exists and otherwise
+  creates a new file. Returns a promise that resolves with the the opened file.
+
+{% Aside 'warning' %} File names are restricted to lowercase alphanumeric characters and underscore
+(`a-z`, `0-9`, `_`). {% endAside %}
+
+{% Aside %} A file can only be opened once. This means concurrent access from different tabs is
+currently not possible. {% endAside %}
+
+- `storageFoundation.delete(name)`: Removes the file with the given name. Returns `undefined`.
+- `storageFoundation.rename(oldName, newName)`: Renames the file from the old name to the new name
+  atomically.  Returns `undefined`.
+- `storageFoundation.getAll()`: Returns a promise that resolves with an array of all existing file
+  names.
+- `storageFoundation.requestCapacity(requestedCapacity)`: Requests new capacity (in bytes) for usage
+  by the current execution context. Returns a promise that resolved with the remaining amount of
+  capacity available.
+
+{% Aside %} The Storage Foundation API achieves fast and predictable performance by implementing its
+own quota management system. Web applications must explicitly ask for capacity before storing any
+new data. This request will be granted according to the browser's quota guidelines. Anytime an
+application starts a new JavaScript execution context (e.g., new tab, new worker, reloading the
+page), it must make sure it owns sufficient capacity before writing new data. {% endAside %}
+
+- `storageFoundation.releaseCapacity(toBeReleasedCapacity)`: Releases unused capacity (in bytes)
+  from the current execution context. Returns a promised that resolves with the remaining amount of
+  capacity available.
+- `storageFoundation.getRemainingCapacity()`: Returns a promise that resolves with the capacity
+  available for the current execution context.
+
+### File handles
+
+Working with files happens via the following functions:
+
+{% Aside %} Storage Foundation API used to be called NativeIO. Some references to this name still
```

Comment:
```suggestion
{% Aside %} The Storage Foundation API used to be called NativeIO. Some references to this name still
```

Reply:
You should move this up so that you can correctly name the interface returned by `window.storageFoundation`.